src/main/java/uk/ac/ebi/subs/validator/coordinator/CoordinatorListener.java

```diff
@@ -185,4 +188,21 @@ public void processFileDeletionRequest(FileDeletedMessage fileDeletedMessage) {
             logger.error("Error handling file deletion for submission (id: {})", submissionID);
         }
     }
+
+    /**
+     * Submittable deletion entry point for triggering a file reference and chained validation
+     * based on the given submission ID..
+     * @param storedSubmittableDeleteMessage contains the ID of the submission to validate
+     */
+    @RabbitListener(queues = SUBMISSION_SUBMITTABLE_DELETED)
+    public void processSubmittableDeletion(StoredSubmittableDeleteMessage storedSubmittableDeleteMessage) {
+        String submissionID = storedSubmittableDeleteMessage.getSubmissionId();
+
+        if (!fileValidationRequestHandler.handleFilesWhenSubmittableChanged(submissionID)) {
```

Comment:
The idea here is not just to log an error, if something is wrong the chained validation is not supposed to be triggered. See `processStudySubmission (...)` for example.